lua/octo/util.lua

```diff
@@ -180,7 +180,7 @@ function M.update_metadata(metadata, start_line, end_line, text)
 end
 
 function M.update_issue_metadata(bufnr)
-  local mark, text, start_line, end_line, metadata
+  local header_mark, body_mark, text, header_line, start_line, end_line, metadata
```

Comment:
This shouldnt be needed neither

Reply:
Closing in regards of [this comment](https://github.com/pwntester/octo.nvim/pull/116#issuecomment-789578727).